content/info/hardware.md

```diff
@@ -0,0 +1,28 @@
++++
+title = "Hardware development"
+date = 2024-01-13
++++
+
+## Contract hardware development
```

Comment:
maybe instead:
Hardware development and consulting

Reply:
```suggestion
## Hardware development and consulting
```